source/patterns/@aws-solutions-constructs/core/lib/fargate-defaults.ts

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Are all these defaults derived from service documentation or vetted with any SME?

Reply:
Met with 2 members of Containers TFC and adjusted values.